packages/gatsby/src/datastore/lmdb/query/common.ts

```diff
@@ -0,0 +1,141 @@
+import { DbComparator, IDbFilterStatement } from "../../common/query"
+import { IGatsbyNode } from "../../../redux/types"
+import { getValueAt } from "../../../utils/get-value-at"
+
+export function isDesc(
+  sortOrder: "asc" | "desc" | "ASC" | "DESC" | boolean | void
```

Comment:
Where is the lowercased version of ASC/DESC coming from? In GraphiQL I only ever saw the uppercased one

Reply:
This was in our types for fast filters. Actually, those types had _only_ `"asc"` and `"desc"` variants. No uppercase versions. I've added uppercase variants but we also use lowercase in tests, e.g.:

https://github.com/gatsbyjs/gatsby/blob/cd4571cfd392db9332337761881a4b60af32468a/packages/gatsby/src/schema/__tests__/run-query.js#L1868-L1873

 Also it is possible if you use `runQuery` in custom resolver:

```js
const result = await nodeModel.runQuery({
  sort: {
    fields: [`foo`],
    order: [`asc`],
  }
})
```